### ISSUE TITLE
when install and config language package zh-CN, I got 404 and language pack not work.

### ISSUE THREAD

**Issue by humbinal** [cdr/code-server#2191]
when I install  language package zh-CN [Chinese (Simplified)], restart code-server and refresh browser page, I got a 404 and language pack not work.

the language package request:
```
Request URL: http://192.168.56.111:8443/%7B%7BBASE%7D%7D/resource/?path=%2Fconfig%2Fdata%2Fclp%2Fb9c1ee9f6b4fe64b286f60e83ae51da7.zh-cn%2Fde41646fc402b968ca6d555fdf2da7de9554d28a%2Fvs!workbench!workbench.web.api.nls.json
Request Method: GET
Status Code: 404 Not Found
```

when I remove the `%7B%7BBASE%7D%7D` from the URL like this:
```
http://192.168.56.111:8443/resource/?path=%2Fconfig%2Fdata%2Fclp%2Fb9c1ee9f6b4fe64b286f60e83ae51da7.zh-cn%2Fde41646fc402b968ca6d555fdf2da7de9554d28a%2Fvs!workbench!workbench.web.api.nls.json
```

I request this URL manually , I found it's OK.

Here is my environmental information:
- Web Browser: Chrome 85 x86_64
- Local OS: Windows10
- Remote OS: CentOS7.7
- Remote Architecture: x86_64
- `code-server --version`: 3.5.0 de41646fc402b968ca6d555fdf2da7de9554d28a

**Comment by GoFly233** [comment]
初步断定是一个 bug
URLEncode `%7B%7BBASE%7D%7D` 可以得到 `{{BASE}}`


我用的是 nginx，然后我在 `location /` 加了一条重定向 `rewrite ^/\{\{BASE\}\}(.*) https://YOUR-CODESERVER-ADDRESS$1 permanent;` 临时解决，Apache 或者 Caddy 应该也可以使用类似的方法重定向


------
Initially, it was a bug.

URLEncode `%7B%7BBASE%7D%7D` will get `{{BASE}}`.

I used nginx, and then I added a redirection rule `rewrite ^/\{\{BASE\}\}(.*) https://YOUR-CODESERVER-ADDRESS$1 permanent;` 
 in `location /` temporary solution, Apache or Caddy should also be able to use similar methods for redirection.

And sorry for my poor English. It's really a serious bug.

**Comment by GoFly233** [comment]
I hope to fix this bug quickly, or I will die. :(

**Comment by code-asher** [comment]
This should be resolved in 3.6.0. Let me know if it doesn't!

**Comment by humbinal** [comment]
This bug has been fixed in 3.6.0, thanks !
